app/rest/controllers/pom.xml

```diff
@@ -42,6 +42,11 @@
       <artifactId>dao</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.syndesis</groupId>
+      <artifactId>jsondb</artifactId>
+    </dependency>
+
```

Comment:
Why is this dependency required here ? jsondb must only be accessed from the DAO package, never directly.

Reply:
I'm not sure we need a DAO right now.  I don't plan on implementing any alternative access methods that would require a common abstraction.

Reply:
Sorry, disagree here. If we couple jsondb like this directly as dependency the DAO abstraction is useless anyway. 

If you find an other way to avoid the dependency here, that's probably fine for me, too. Only 'rest/runtime' is allowed to depend on jsondb.